testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +962,18 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+def test_cryst_em_warning():
+    #issue 2599  
+    with pytest.warns(UserWarning) as record:
+        u = mda.Universe(PDB_CRYOEM_BOX)
+    assert record[0].message.args[0] == "1 A^3 CRYST1 record," \
+                                      " its usually a placeholder for" \
```

Comment:
Getting a flake8 warning here, can you shift these lines so that the " of each line is aligned (assuming it doesn't break 80 characters per line).